library/src/scala/Tuple.scala

```diff
@@ -183,7 +183,6 @@ object Tuple {
     case (h1 *: t1, h2 *: t2) => (h1, h2) *: Zip[t1, t2]
     case (EmptyTuple, _) => EmptyTuple
     case (_, EmptyTuple) => EmptyTuple
```

Comment:
How about

```suggestion
    case _ => EmptyTuple
```

Reply:
Great point! and it matches up with the doc too